pkg/cmd/cmd.go

```diff
@@ -65,6 +66,14 @@ func (r *Runner) RunWithStdin(ctx context.Context, cwd string, env []string, std
 	cmd.Stdout = outbuf
 	cmd.Stderr = errbuf
 	cmd.Stdin = bytes.NewBufferString(stdin)
+	if v := ctx.Value("syncSig"); v != nil {
```

Comment:
I initially said I was OK with this, but when I apply this and do manual testing, I *want* ^C to kill git.  So I don't think we should do this part.

Reply:
ok.